Python-Resources.md

```diff
@@ -0,0 +1,32 @@
+# Python Resources
+
+[Home](Python)
+
+## Tutorials
+
+* [Google Python Class](https://developers.google.com/edu/python/)
+* [Python Official Tutorials](https://docs.python.org/3/tutorial/)
+* [Learn Python the Hard Way](http://learnpythonthehardway.org/book/)
+* [Think Python](http://www.greenteapress.com/thinkpython/html/index.html)
+* [Learn Python in X minutes](https://learnxinyminutes.com/docs/python/)
+* [Learn Python 3 in X minutes](https://learnxinyminutes.com/docs/python3/)
+* [Dive into Python 3](http://www.diveintopython3.net/)
+* [The Hitchhiker's Guide to Python](https://python-guide.readthedocs.org/en/latest/)
```

Comment:
This is too good a document to be this down. I would recommend putting it second.